### DISCUSSION TITLE
bash files being changed

### DISCUSSION THREAD

**Discussion by simon-blum-cko** [lindell/multi-gitter#367] [Q&A] [answered]
Hi, 

I'm running with script for an org to add a workflow to each repo. it's changing the metadata of all .sh files in the repo without changing any content. any idea why this is? 

[Edit] this is run on a windows machine and may be due to that, however it needs fixing. any idea what i can do? 

Thanks

**Comment by lindell** [comment]
What kind of metadata are we talking about?
Does the script change these files?
Does this happen for all bash files on all repositories? If not, can you create an example repo where this can be reproduced. 
Does it also happen when running with `--git-type=cmd`.

**Comment by simon-blum-cko** [comment]
Here's an example of the changes made: 
![image](https://github.com/lindell/multi-gitter/assets/89977041/800f0f5d-35e4-492b-866b-412ee6ca420f)
We believe it represents a change to line endings due to running the tool on windows. However all the fixes i've found online i can't execute as the git cmd's are being run by multigitter so i cant run my own commands. Any idea / have you seen this before?

**Reply by lindell** [reply, marked as answer]
It seems that the **permissions** are being changed. 

Does the script you are running change these files? Or touch them in any kind of way? Does the same occur if the script you run does nothing?
Does it also happen when running with `--git-type=cmd`

**Reply by simon-blum-cko** [reply]
thanks, running with this flag fixed the issue

**Reply by lindell** [reply]
Great that it worked 😄 But can you still provide the context I requested and a way to reproduce this so it can be fixed?

**Reply by lindell** [reply]
@simon-blum-cko 
Any progress in providing a way to repro this?
